transform/mattermost-analytics/models/intermediate/product/boards_active_users/int_boards_active_days_spined.sql

```diff
@@ -0,0 +1,47 @@
+{{
+    config({
+        "materialized": "table",
+        "cluster_by": ['activity_date', 'server_id'],
+        "unique_key": ['activity_date', 'server_id', 'user_id'],
+        "snowflake_warehouse": "transform_l"
+    })
+}}
+with user_first_active_day as (
+    select
+        server_id,
+        user_id,
+        min(activity_date) as first_active_day
+    from
+        {{ ref('int_boards_active_days') }}
+    where
+        activity_date >= '{{ var('telemetry_start_date')}}'
+    group by 1, 2
+), spined as (
+    -- Use date spine to fill in missing days
+    select
+        first_day.server_id,
+        first_day.user_id,
+        all_days.date_day
+    from
+        user_first_active_day first_day
+        left join {{ ref('telemetry_days') }} all_days on all_days.date_day >= first_day.first_active_day
+)
+select
+    cast(spined.date_day as date) as activity_date
+    , spined.server_id
+    , spined.user_id
+    , coalesce(user_active_days.is_active, false) as is_active_today
+    , max(is_active_today) over(
+        partition by spined.user_id order by spined.date_day
+        rows between 6 preceding and current row
+    ) as is_active_last_7_days
+    , max(is_active_today) over(
+        partition by spined.user_id order by spined.date_day
+        rows between 29 preceding and current row
+    ) as is_active_last_30_days
+from
+    spined
+    left join {{ ref('int_boards_active_days') }} user_active_days
+        on spined.date_day = user_active_days.activity_date
```

Comment:
Wondering if it makes sense to use a similar approach like in `int_server_active_days_spines.sql`  (also to be consistent across products) and also include the server(telemetry) data (besides the event/activity/tracks one).

Reply:
This is a good idea. There's a risk that current implementation "hides' some data. @catalintomai the PR has been updated to use a proper spine.

Since this is starting to become a common pattern, it might make sense to see if we can somehow streamline this process.